pegasus/sites.v3/hourofcode.com/public/learn/index.haml

```diff
@@ -21,6 +21,11 @@ social:
     }
   }
 
+:css
+  .noFocusButton:focus {
+    outline: 0;
+  }
```

Comment:
Unfortunately you can't style psuedo classes (in this case, :focus) in inline CSS.  Lots of discussion in React community about this, but short of adding another framework this seems easiest way.